endoreg_db/models/media/pdf/raw_pdf.py

```
@@ -750,8 +750,17 @@ def get_report_reader_config(self):
         return settings_dict
 
     @staticmethod
-    def get_pdf_by_pk(pk: int) -> "RawPdfFile":
+    def get_report_by_pk(pk: int) -> "RawPdfFile":
         try:
             return RawPdfFile.objects.get(pk=pk)
         except RawPdfFile.DoesNotExist:
             raise ValueError(f"report with ID {pdf_id} does not exist.")
```

Pylint (4.0.4)

[error] Undefined variable 'pdf_id'

(E0602)